### PR TITLE
SCALCE-2.7 easyconfigs for goolf and ictce

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 1 out of 9
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/ed51191ed636627e6187 for a full test report.

**Comment by fgeorgatos** [comment]
@pescobar: this indeed is almost duplicate of #1242, 
yet we stand a chance to have here a better PR (ref. @boegel 's remarks)

File: easybuild/easyconfigs/s/SCALCE/SCALCE-2.7-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ is the same throughout the file. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(namelower)s-v%(version)s-src.tar.gz']
+
+# firt move to folder zlib-1.2.8 to run make and generate libz.a
+# then run make in the parent folder
+premakeopts = "cd zlib-1.2.8 && make && cd .. && "
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/SCALCE/SCALCE-2.7-goolf-1.4.10.eb]: minor changes needed here, put this in REVIEW plz!

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by other PR.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**